web/src/components/profile/ProfileNewJerseyAddress.tsx

```diff
@@ -34,7 +38,7 @@ export const ProfileNewJerseyAddress = (): ReactElement => {
   const { setIsValid: setIsValidZipCode } = useFormContextFieldHelpers("addressZipCode", ProfileFormContext);
 
   useMountEffectWhenDefined(() => {
-    if (business && isOwningBusiness(business)) {
+    if (business && (isStartingBusiness(business) || isOwningBusiness(business))) {
```

Comment:
pebble - given that this component renders on starting and owning profile pages, i don't think the conditional is needed.

Reply:
Not sure how this ended up back in. We had deleted it. TY!